pkg/search/postgres/store.go

```diff
@@ -171,10 +172,16 @@ func (s *genericStore[T, PT]) Exists(ctx context.Context, id string) (bool, erro
 }
 
 // Count returns the number of objects in the store.
-func (s *genericStore[T, PT]) Count(ctx context.Context) (int, error) {
+func (s *genericStore[T, PT]) Count(ctx context.Context, q *v1.Query) (int, error) {
 	defer s.setPostgresOperationDurationTime(time.Now(), ops.Count)
```

Comment:
Would keeping the metrics under index op communicate correctly? IMO it would be wrong to tie this back to index operation. 
Does this change necessitate a customer-facing deprecation? Do you have any proposal to translate the existing index operations metrics to the postgres op metrics?

Reply:
I think its fine to just change the metric